include/net/net_pkt.h

```diff
@@ -526,6 +557,23 @@ static inline void net_pkt_set_ipv6_hop_limit(struct net_pkt *pkt,
 }
 #endif /* CONFIG_NET_IPV6 */
 
+static inline u16_t net_pkt_ip_opts_len(struct net_pkt *pkt)
+{
+#if defined(CONFIG_NET_IPV6)
+	return pkt->ipv6_ext_len;
+#endif
+
+#if defined(CONFIG_NET_IPV4)
+	return pkt->ipv4_opts_len;
+#endif
```

Comment:
Compiler will probably warn if both ipv4 and ipv6 are enabled here. You'll need a #else etc...

Reply:
I checked it with echo_server sample. No warning.

Reply:
Weird, because the ipv4 part is obvious dead code if ipv6 part is enabled

Reply:
ok, coverity might through, I will modify it.

Reply:
changed.